pontoon/contributors/static/js/settings.js

```diff
@@ -1,4 +1,118 @@
 $(function () {
+  $('.field .input')
+    .unbind('keydown.pontoon')
+    .bind('keydown.pontoon', function (e) {
+      if (e.which === 13) {
+        $(this).trigger('blur');
+        return false;
+      }
+    });
+
+  $('.field .input').on('blur', function (e) {
+    e.preventDefault();
+
+    const self = $(this);
+    const value = self.val().trim();
+    const attribute = self.data('attribute');
+    const originalValue = self.data('original-value') || '';
+
+    if (value === originalValue) {
+      return;
+    }
+
+    $.ajax({
+      url:
+        '/user/' +
+        $('#profile input[name="username"]').val() +
+        '/attributes/field/',
+      type: 'POST',
+      data: {
+        csrfmiddlewaretoken: $('body').data('csrf'),
+        attribute: attribute,
+        [attribute]: value || '',
```

Comment:
`value` is set from a `.trim()` result, so if it's falsy, it must be `''`.
```suggestion
        [attribute]: value,
```